install/installer/cluster_dev.py

```diff
@@ -110,26 +118,55 @@ def validate(self, document=None, interactive=True):
             )  # Move cursor to end
 
 
-def ask_user(question_name, non_interactive_value=None):
+@typechecked
+class AWSUserNameValidator(Validator):  # pylint: disable=too-few-public-methods
+    """Validate user input"""
+
+    def validate(self, document=None, interactive=True):
+        """
+        Validate user input string to AWS username restrictions
```

Comment:
Usually, the title goes to the first line as long as it fits 72 chars length restriction:
```suggestion
        """Verify that the user input string conforms AWS username requirements.
```

Reply:
Okay, can I use 100 chars instead of 72?

Reply:
No, pydocstyle specifically sets 72 chars for the title line. The following paragraph can be up to 79 as per PEP 8. Also, short lines improve readability and accessibility (i.e. folks with large fonts and interfaces with restricted width like PR split-view diff on GH will benefit from it a lot).

If you can't fit it in 72 chars, this probably means that your function does way more than it should and must be split into separate functions.